apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx

```diff
@@ -47,12 +48,12 @@ export default async function Checkout({ params }: { params: CheckoutParams }) {
     <section>
       {!session && (
         <>
-          <h4 className="font-semibold text-grey-600 text-lg mt-10">
+          <h2 className="font-semibold text-grey-600 text-lg mt-10">
```

Comment:
Not sure it matters, but the styling on this `<h2>` is _slightly_ different than the 2 below:

```sh
git grep -n -A1 "<h2" -- apps | grep "className=[\"'].*[\"']"

apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx:51:          <h2 className="font-semibold text-grey-600 text-lg mt-10">
apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx-98-              className="font-semibold text-grey-600 text-lg mt-14 mb-5"
apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx-108-             className="font-semibold text-grey-600 text-lg mt-14 mb-5"
```

```diff
+ mt-10
- mt-14 mb-5
```

Reply:
This was copied from existing payments-server. I'd rather keep it as is for now.